scripts/partition_manager/partition_manager.rst

```diff
@@ -718,10 +718,10 @@ Static configuration
 ********************
 
 By default, the Partition Manager dynamically places the partitions in memory.
-However, if you have a deployed product that consists of multiple images, where only a subset of the included images can be upgraded through a firmware update mechanism, the upgradable images must be statically configured.
```

Comment:
Here we remove the general statement ` the upgradable images must be statically configured`, and leave the reader to infer the limitation from the example following this.
I think we should still give a statement that in such a case, upgradable images must have partitions that are static/matches the bootloader.

Reply:
New phrasing:
> By default, the Partition Manager dynamically places the partitions in memory.
However, there are cases where a deployed product can consist of multiple images, and only a subset of these images can be upgraded through a firmware update mechanism.
In such cases, the upgradable images must have partitions that are static or are matching the bootloader partition map.
For example, if a device includes a non-upgradable first-stage bootloader and an upgradable application, the application image to be upgraded must be linked to the same address as the one that is deployed.
For this purpose, the Partition Manager provides the static configuration to define static partitions.

Reply:
How about `In such cases, the upgradable images must have partitions that are static in order to match the bootloaders partition map.`
?